src/project_types.rs

```diff
@@ -1,17 +1,38 @@
 /// Structure to represent a Proton Project.
 /// This is what will be written to a Protonfile at the project root.
-#[derive(Debug, RustcEncodable, RustcDecodable)]
+
```

Comment:
```
    /**
     * Merges stuff
     * @param dataIds
     */
    @Override
    public String runMerge(List<String> dataIds) throws TException {
```

Idk, it looks normal to me to not have a line there. Well, do it however you want. Maybe we should run [rustfmt](https://github.com/rust-lang-nursery/rustfmt) on it?